addons/website/static/src/core/colibri.js

```diff
@@ -82,7 +82,7 @@ export class Colibri {
             }
             for (const cl in value) {
                 for (const c of cl.trim().split(" ")) {
-                    el.classList.toggle(c, value[cl]);
+                    el.classList.toggle(c, value[cl] || false);
```

Comment:
classList.toggle requires a boolean? didn't know that. this commit needs a test then :)